scripts/core-testing.sh

```diff
@@ -105,10 +105,10 @@ if [ -z "${GITHUB_ACTIONS}" ] || [ "$1" == "core_prep" ] ; then
 		fi
 		cd "${coredir}" || exit
 		echo ""
-		echo " ** Resetting to dev **"
+		echo " ** Resetting to master **"
```

Comment:
While I do understand this caused a recent issue, we should align with dev, not master. It's better to wait a little with a release + also we won't know that something breaks this way up front and always will find out after the fact.

Reply:
Added PR to for this PR-branch with suggestions

Reply:
PR has been merged.